examples/collatz/weaver.toml

```diff
@@ -2,6 +2,9 @@
 binary = "./collatz"
 rollout = "5m"
 
+[multi]
+mtls = true
```

Comment:
Feel free to leave it to `false` by default.

Reply:
Done.